packages/dotcom-build-code-splitting/src/__test__/index.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { PageKitCodeSplittingPlugin } from '../index'
+import webpack from 'webpack'
+import path from 'path'
+
+describe('Code Splitting', () => {
+  it('chunk privacy modules', async () => {
+    await new Promise((resolve) =>
+      webpack(
+        {
+          mode: 'none',
+          entry: {
+            scripts: path.join(__dirname, '/__fixtures__', 'entry-point.js')
+          },
+          output: {
+            filename: '[name].js',
+            path: path.join(__dirname, '/tmp')
+          },
+          plugins: [new PageKitCodeSplittingPlugin()]
+        },
+        function (err, stats) {
```

Comment:
```suggestion
        function (error, stats) {
```

In line with the [style guidelines](https://github.com/Financial-Times/dotcom-page-kit/blob/master/contribution.md#user-content-general-style) for this repo, I would suggest changing `err` here and on the following lines to `error`.